### PR TITLE
Image 404 template fix

### PR DESCRIPTION
**Pull request by PBruk**
Tagging @frrrances for code review: this branch adjusts the page structure for several templates, including the 404 page, as well as the image.php file in addition to their styling.

### CODE REVIEW

File: css/partials/_layout.scss [hunk 1]
```diff
@@ -263,4 +263,9 @@ html.no-js .js-hidden {
 	@include bp-tablet--portrait {
 		display: none;
 	}
+}
+
+.content {
+    margin-top: 2em;
```
Comment (frrrances) [inline css/partials/_layout.scss]: Should this be rem, percent, or px so it remains stable?

File: single.php [hunk 2]
```diff
@@ -1,32 +1,50 @@
 <?php
 /**
- * The Template for displaying all single posts.
  *
- * @package MIT_Libraries_Parent
- * @since 1.2.1
+ * This is the template that displays single posts
```
Comment (frrrances) [inline single.php]: I think we want to leave the header info as MIT_Libraries_Parent.

File: single.php [hunk 3]
```diff
@@ -1,32 +1,50 @@
...
-	</div><!-- #primary -->
+get_header(); ?>
 
-<?php get_sidebar(); ?>
-<?php get_footer(); ?>
+<?php get_template_part('inc/breadcrumbs', 'child') ?>
+	
+<div id="stage" class="inner" role="main">
+
+	<?php get_template_part('inc/postHead'); ?>
```
Comment (frrrances) [inline single.php]: Why postHead here? I don't see that in the other templates...

File: single.php [hunk 4]
```diff
@@ -1,32 +1,50 @@
 <?php
```
Comment (frrrances) [inline single.php]: Why did this file change so much? Is it a page we're not using? If so, should it be removed or have a note at the top of the file?

**Comment by frrrances** [comment]
A couple small changes/questions - once those are addressed, we can put this on test for review.

File: single.php (hunk 4 above)
Comment (PBruk) [inline single.php]: This template changed to match the streamlined structure of our other templates, it included lots of code that we no longer used, which I removed. Also, his template is mostly used on Child sites as the main site doesn't have many posts - if any.

File: single.php (hunk 3 above)
Comment (PBruk) [inline single.php]: postHead was added here because it is inherited by Child sites which require it.

File: single.php (hunk 2 above)
Comment (PBruk) [inline single.php]: Reverted, added new version, as well, 1.4.0

File: css/partials/_layout.scss (hunk 1 above)
Comment (PBruk) [inline css/partials/_layout.scss]: Changed to px.

**Comment by frrrances** [comment]
closing in favor of #102

**Pull request closed without merging**